cronograma/r_dev_day/RDevDay-pt.qmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "R Dev Day"
+knitr:
+  opts_chunk:
+    echo: false
+    message: false
+    warning: false
+editor: 
+  markdown: 
+    wrap: sentence
+---
+
+# R Dev Day \@ LatinR 2024
+
+**Datas:** 18 e 19 de novembro de 2024, das 17h30 às 20h (UTC-3)
+
+**Modo:** Online
+
+**Inscrição:** <https://pretix.eu/r-contributors/r-dev-day-latinr2024/>
+
+**Idiomas:** Inglês, português e espanhol
+
+# Sobre
+
+O R Dev Day \@ LatinR 2024 é um evento colaborativo no qual os participantes trabalharão em pequenos grupos para contribuir para o R básico ou para a infraestrutura que dá suporte a essas contribuições, por exemplo
+
+-   Traduzir strings de mensagens para espanhol/português brasileiro
+-   Propor melhorias nos arquivos de ajuda do R
+-   Analisar um erro relatado no R
+-   Desenvolver uma correção para um erro relatado no R
+-   Adicionar conteúdo ao [R Dev Guide](https://contributor.r-project.org/rdevguide/)
```

Comment:
```suggestion
-   Adicionar conteúdo ao [R Dev Guide](https://contributor.r-project.org/rdevguide/), o guia de contribuição ao R base
```

Reply:
Apply similar update to English:

-   Add content to the [R Dev Guide](https://contributor.r-project.org/rdevguide/), the guide for contributors to base R